insights/parsers/ceph_version.py

```diff
@@ -124,6 +124,10 @@
     "18.2.1-262": {'version': "7.1.2", 'major': '7', 'minor': '1', 'downstream_release': '2'},
     "19.2.0-53": {'version': "8.0", 'major': '8', 'minor': '0', 'downstream_release': '0'},
     "19.2.0-55": {'version': "8.0.1", 'major': '8', 'minor': '0', 'downstream_release': '1'},
+    "19.2.0-98": {'version': "8.0.1", 'major': '8', 'minor': '0', 'downstream_release': '2'},
+    "19.2.0-124": {'version': "8.0.1", 'major': '8', 'minor': '0', 'downstream_release': '3'},
+    "19.2.0-137": {'version': "8.0.1", 'major': '8', 'minor': '0', 'downstream_release': '4'},
```

Comment:
IDKM, just, is this a typo in the `'version': "8.0.1"` value? say, should be `'version': "8.0.4"` here?

Reply:
@JoySnow - updated, please checkl